db/migrate/20220916164800_create_members.rb

```diff
@@ -0,0 +1,13 @@
+class CreateMembers < ActiveRecord::Migration[6.1]
+  def change
+    create_table :members do |t|
+      t.string :name, null: false
+      t.string :facebookUrl
```

Comment:
In Ruby on Rails exist a naming convention that you have to use `snake_case` for database field names. Instead of `facebookUrl` it is correct the use of `facebook_url`

Reply:
Resolved with snake_case_fix migration